### ISSUE TITLE
Unable to run android demo with custom model

### ISSUE THREAD

**Issue by AlexRibard** [microsoft/onnxruntime#9248]
Hi all!

I am trying to profile ORT on a mobile device for a custom onnx model.
Here is the following steps I am following:

* Step 1:
I have downloaded the following tutorial:
https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples/image_classification/android
The tutorial runs. 

Now since I am interested in inference speed only,
I have commented out :
```
percentMeter.progress = (result.detectedScore[0] * 100).toInt()
detected_item_1.text = labelData[result.detectedIndices[0]]
detected_item_value_1.text = "%.2f%%".format(result.detectedScore[0] * 100)

if (result.detectedIndices.size > 1) {
    detected_item_2.text = labelData[result.detectedIndices[1]]
    detected_item_value_2.text = "%.2f%%".format(result.detectedScore[1] * 100)
}

if (result.detectedIndices.size > 2) {
    detected_item_3.text = labelData[result.detectedIndices[2]]
    detected_item_value_3.text = "%.2f%%".format(result.detectedScore[2] * 100)
}
``` 
In `private fun updateUI(result: Result)` to just output the `processTimeMs`.

* Step 2:
Now I convert my onnx file to an `.ort` file like follows:
```
python .\tools\python\convert_onnx_models_to_ort.py .\models\
```
That generates me the `.ort` model aswell as a `required_operators.config` file.

* Step 3:
I add the `.ort` in `onnxruntime-inference-examples\mobile\examples\image_classification\android\app\src\main\res\raw` and edit the name in `readModel()`

Not sure were to add the config file as I do not see any in the project structure.

* Step 4: 
Projects runs but I get the following error:
```
ai.onnxruntime.example.imageclassifier E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ai.onnxruntime.example.imageclassifier, PID: 26742
    ai.onnxruntime.OrtException: Error code - ORT_FAIL - message: inference_session.cc:1217 AssignNodesToEpsFromHashesImpl Failed to find kernel def hash (9018732444147417520) in kernel registries for MaxPool(1) node with name 'squeezenet0_pool0_fwd'.
 ```

My guess is that it fails to load the specific ops of the model as the config file is not specified, but I don't know how to specify it.
Any help would be greatly appreciated!

Thanks

**Comment by AlexRibard** [comment]
*TLDR:* I am trying to run a custom model using the provided android demo project, but am getting the following error
```
ai.onnxruntime.example.imageclassifier E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ai.onnxruntime.example.imageclassifier, PID: 26742
    ai.onnxruntime.OrtException: Error code - ORT_FAIL - message: inference_session.cc:1217 AssignNodesToEpsFromHashesImpl Failed to find kernel def hash (9018732444147417520) in kernel registries for MaxPool(1) node with name 'squeezenet0_pool0_fwd'.
```
Here is the model in question
[squeezenet1.1.zip](https://github.com/microsoft/onnxruntime/files/7269959/squeezenet1.1.zip)

**Comment by gwang-msft** [comment]
Your `squeezenet1.1` model is using ONNX Opset 7, the pre-built ORT-Mobile package only supports Opset 12 and 13, see https://onnxruntime.ai/docs/reference/mobile/prebuilt-package/1.8%20ORTMobilePackageOperatorTypeSupport

You can try one of the following,
- Update the Opset of you ONNX model, see, https://github.com/onnx/onnx/blob/master/docs/VersionConverter.md, please note, the conversion tool is very limited
- If you train the model yourself, you may try to re-convert the original framework model to ONNX using Opset 12 or 13, for example, specify `opset_version` in `torch.onnx.export` if you are using PyTorch
- Or you can try to custom build ONNX Runtime for Android to support your `squeezenet1.1` model, see https://onnxruntime.ai/docs/tutorials/mobile/custom-build.html

**Comment by AlexRibard** [comment]
Ok thanks for the info.
Would it be possible to throw a warning when using `convert_onnx_models_to_ort`?

**Comment by skottmckay** [comment]
See https://onnxruntime.ai/docs/reference/mobile/helpers.html for info on some new helpers to do model analysis when creating the ORT format model. Also contains a helper to update the onnx opset.
